### ISSUE TITLE
Infra work for bulletproof Apple account usage

### ISSUE THREAD

**Issue by autonome** [ipfs-shipyard/ipfs-desktop#1492]
Recommendation from Andrew Schwab after confirming Twilio numbers are detected:

> * Create a Google Group (I'd recommend something like ipfs-apple-id@)
> * Create an Apple ID with that group email
> * Document the Apple ID email login and reset question in 1Password in a shared vault
> * Setup the phones for members of the group for SMS 2FA
> * Add a note in 1Password indicating whose phones are currently connected

So we need to determine if we need to create new Apple ID or switch existing account @olizilla has created to use the new email alias.

From my previous iOS work, I think it depends on how @olizilla set the account up. A real pain to move stuff between accounts, and accounts have a lot of personal detail and config (at least in the US).

**Comment by jessicaschilling** [comment]
@rafaelramalho19 -- can you please prioritize this over the loading indicator work? Thank you!

**Comment by rafaelramalho19** [comment]
I've followed the steps but apparently there was a problem creating the account in the last steps 😕 

![image](https://user-images.githubusercontent.com/24696635/81385048-9d1f5c00-910a-11ea-9f73-06ef2e5d0e39.png)

**Comment by andyschwab** [comment]
Followup: looking into solving this using Apple Developer Program for Organizations capabilities.

**Comment by jessicaschilling** [comment]
@andyschwab -- just checking in on what you've learned? Thanks!

**Comment by andyschwab** [comment]
@jessicaschilling - identified the solid path forward, but currently on hold until we can use the best account to setup a Developer Program Organization and update permissions accordingly. I'll continue to track this and action as soon as it is unblocked.

**Comment by jessicaschilling** [comment]
Awesome, thanks @andyschwab! I'll move out of our in-progress queue until you tell me otherwise.

**Comment by lidel** [comment]
I believe this is done, migrated new secrets from Travis to Github Actions.
